src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -15,9 +14,10 @@ const GovernanceReviewTeam = () => {
   const isUserSet = useSelector((state: AppState) => state.auth.isUserSet);
   const flags = useFlags();
 
+  // TODO: this is causing the Routes to unexpectedly unmount
```

Comment:
`RenderPage` is continuously unmounted and remounted on every render because it's declared inside of a functional component. This is not good. However this side effect fixes bug(s) we have in the application.

One example is adding GRT/GRB meeting dates. When you save those dates, you're redirected back to the main gov team hub. **That does not trigger a refetch to get the updated data with the new GRT/GRB dates**.

Since the routes are unmounted/remounted, that forces a re-fetch. This explains why things have been working, but it's something we should fix. We should make sure things are tested before we refactor any further.